err.c

```diff
@@ -45,15 +49,37 @@ static void fdprintf(int fd, const char *fmt, ...)
 
 extern const char *__progname;
 
+static void vwarnsyslog(int errcode, const char *fmt, va_list vl)
+{
+	char newfmt[BUFSIZ];
+
+#define suffix ": %m"
```

Comment:
I don't live in C but it seems weird that you'd want to use a macro instead of a variable here?

Reply:
I wanted to avoid breaking sizeof, but now that I think about it I can use a flexible array here.